src/site/content/en/blog/advancing-framework-ecosystem-cds-2019/index.md

```diff
@@ -0,0 +1,208 @@
+that:
+
++   72% of participants in the
+    [MDN's First Annual Web Developer & Designer Survey](https://hacks.mozilla.org/2019/12/presenting-the-mdn-web-developer-needs-assessment-web-dna-report/)
+    use at least one of these frameworks and libraries.
++   Over
+    [320,000 sites](https://bigquery.cloud.google.com/savedquery/1086077897885:24ffb259f2a04a7f9955e44f6e0298e9)in
+    the top 5 million URLs analyzed by HTTP Archive use at least one of these frameworks and libraries.
++   When grouped by time spent, 30 of the top 100 URLs use at least one of these frameworks and
+    libraries. (Research was done on internal data.)
+
+This means that **better open-source tooling can directly result in a better web** and that's why
+Chrome engineers have started working directly with external framework and library authors.
+
+## Contributions to web frameworks
+
+Frameworks commonly used to build and structure web pages fall into two categories:
+
++   **UI frameworks** (or libraries), such as Preact, React, or Vue, which provide control
+    over an application's view layer (through a component model for example).
++   **Web frameworks**, such as Next.js, Nuxt.js, and Gatsby, which provide an end-to-end system
+    with opinionated features built-in, such as server-side rendering. These frameworks usually
+    leverage a UI framework or library for the view layer.
+
+![A spectrum of UI frameworks and libaries versus Web frameworks](./framework-types.png)
+
+Developers can choose not to use frameworks but by piecing together a view layer library, router,
+styling system, server renderer and so forth, they often end up creating their own type of a
+framework. Although opinionated, web frameworks take care of many of these concerns by default.
+
+The rest of this post highlights many improvements that have recently landed in different frameworks
+and tools, including contributions from the Chrome team. 
+
+## Angular
+
+The Angular team has shipped a number of improvements to version 8 of the framework: 
+
++   [Differential loading](https://angular.io/guide/deployment#differential-builds) by
+    default to minimize unneeded polyfills for newer browsers.
+
+<figure class="w-figure w-figure--center">
+  <img src="differential-loading-angular.png" class="w-screenshot" alt="Graph showing bundle size reduction of angular.io with and without differential builds">
+  <figcaption class="w-figcaption">
+    Bundle size reduction for angular.io with differential builds (from <a href="https://blog.angular.io/version-8-of-angular-smaller-bundles-cli-apis-and-alignment-with-the-ecosystem-af0261112a27">"Version 8 of Angular"</a>)
+  </figcaption>
+</figure>
+
+Bundle size reduction for angular.io with differential builds
+(from[" Version 8 of Angular"](https://blog.angular.io/version-8-of-angular-smaller-bundles-cli-apis-and-alignment-with-the-ecosystem-af0261112a27))
```

Comment:
Looks like this caption was accidentally duplicated so need to delete lines 85 & 86

![image](https://user-images.githubusercontent.com/4713486/72032926-a7f1df00-3245-11ea-9d8c-24dcf99d26d2.png)